src/main/java/com/bunq/sdk/model/generated/object/LabelMonetaryAccount.java

```diff
@@ -117,4 +124,15 @@ public void setBunqMe(MonetaryAccountReference bunqMe) {
     this.bunqMe = bunqMe;
   }
 
+  /**
+   * Whether or not the monetary account is light.
```

Comment:
On this line and the comment above, space between `monetary` and `account`. For consistency, should it be `MonetaryAccount`?

Reply:
generated